### DISCUSSION TITLE
/making-mkdocs-tables-look-like-github-markdown-tables/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#197] [Announcements]
Making mkdocs tables look like Github markdown tables
        
[https://blog.ktz.me/making-mkdocs-tables-look-like-github-markdown-tables/](https://blog.ktz.me/making-mkdocs-tables-look-like-github-markdown-tables/)

**Comment by timlepes** [comment]
First Comment, fools!  Ha!  In like LIGHTNING and IN YOUR FACE!  Yes.  Lightning in your FACE.  WOOOOOOOT!

Just goofing.  Love the Self Hosted show!  Checking out your blog...

**Comment by lonelymaw** [comment]
Looks great, thank you for sharing! :)
